### ISSUE TITLE
Pops and crackles when using pulseaudio

### ISSUE THREAD

**Issue by tremby** [badaix/snapcast#785]
**Describe the bug**

After stopping music I'm hearing pops and crackles coming from snapclient, when running in pulse player mode. I do not get these pops and crackles in the default mode.

**Steps to Reproduce**

1. Start snapclient with `--player pulse`.
2. Play music for a few seconds. It seems to make a difference if it was only quiet or loud; play some loud stuff for a second or two.
3. Stop or pause music.
4. Listen closely and watch `pavucontrol` or similar.
5. Hear and see little pops and crackles intermittently. Whereas the ALSA playback entry would have disappeared after a few seconds, the pulse one is still there indefinitely.

**Environment details**

- OS: Ubuntu x64
- Snapcast version 0.23.0
- Installed from release package on Github

**Log**

You can see where I start and stop music. The crackles start at about where it says 16-47-07 and recur once every second or two.

```log
2021-02-02 16-46-39.997 [Info] (Connection) Resolving host IP for: hoard
2021-02-02 16-46-40.000 [Info] (Connection) Connecting
2021-02-02 16-46-40.000 [Notice] (Connection) Connected to 10.0.0.46
2021-02-02 16-46-40.000 [Info] (Connection) My MAC: "30:5a:3a:57:2c:5d", socket: 8
2021-02-02 16-46-40.066 [Debug] (Connection) outstanding async_write
2021-02-02 16-46-40.067 [Info] (Controller) ServerSettings - buffer: 1000, latency: 0, volume: 100, muted: 0
metadata:{"STREAM":"default"}
2021-02-02 16-46-40.067 [Info] (Controller) Codec: flac, sampleformat: 44100:16:2
2021-02-02 16-46-40.067 [Info] (Player) Player name: pulse, device: default, description: Let PulseAudio server choose the device, idx: 1, sharing mode: unspecified, parameters: <none>
2021-02-02 16-46-40.067 [Info] (Player) Mixer mode: software, parameters: <none>
2021-02-02 16-46-40.067 [Info] (Player) Sampleformat: 44100:16:2, stream: 44100:16:2
2021-02-02 16-46-40.067 [Info] (PulsePlayer) Using buffer_time: 80 ms
2021-02-02 16-46-40.068 [Debug] (PulsePlayer) State changed 2: authorizing
2021-02-02 16-46-40.074 [Debug] (PulsePlayer) State changed 3: setting name
2021-02-02 16-46-40.087 [Debug] (PulsePlayer) State changed 4: ready
2021-02-02 16-46-40.088 [Debug] (Player) setVolume exp with base 10: 1 => 1
2021-02-02 16-46-40.089 [Info] (Stream) No chunks available
2021-02-02 16-46-40.101 [Info] (Controller) diff to server [ms]: 27088.7
2021-02-02 16-46-41.004 [Info] (Stream) No chunks available
2021-02-02 16-46-42.003 [Info] (Stream) No chunks available
2021-02-02 16-46-43.005 [Info] (Stream) No chunks available
2021-02-02 16-46-44.020 [Info] (Stream) No chunks available
2021-02-02 16-46-45.005 [Info] (Stream) No chunks available
2021-02-02 16-46-46.030 [Info] (Stream) No chunks available
2021-02-02 16-46-47.003 [Info] (Stream) No chunks available
2021-02-02 16-46-48.025 [Info] (Stream) No chunks available
2021-02-02 16-46-49.025 [Info] (Stream) No chunks available
2021-02-02 16-46-50.009 [Info] (Stream) No chunks available
2021-02-02 16-46-51.057 [Debug] (Stream) Silent frames: 946, frames: 1246, age: -21.471
2021-02-02 16-46-51.089 [Debug] (Stats) Chunk: 0	0	0	0	1	45	0
2021-02-02 16-46-52.029 [Debug] (Stats) Chunk: 0	0	0	0	34	48	0
2021-02-02 16-46-53.016 [Debug] (Stats) Chunk: -2	-1	0	0	69	44	0
2021-02-02 16-46-54.013 [Debug] (Stats) Chunk: -2	-2	-1	-1	104	45	0
2021-02-02 16-46-55.030 [Debug] (Stats) Chunk: -2	-2	-2	-2	140	45	2
2021-02-02 16-46-56.016 [Debug] (Stats) Chunk: 0	0	-2	-1	175	44	4
2021-02-02 16-46-57.034 [Debug] (Stats) Chunk: -1	0	-1	-1	211	48	4
2021-02-02 16-46-58.003 [Debug] (Stats) Chunk: 0	-1	0	-1	245	57	2
2021-02-02 16-46-59.003 [Debug] (Stats) Chunk: 0	0	0	-1	280	45	0
2021-02-02 16-46-59.651 [Info] (Stream) Exception: Not enough frames available, requested frames: 1246, available: 16
2021-02-02 16-46-59.666 [Debug] (Stream) age > 0: 27ms, dropping old chunks
2021-02-02 16-47-00.004 [Info] (Stream) No chunks available
2021-02-02 16-47-01.005 [Info] (Stream) No chunks available
2021-02-02 16-47-02.006 [Info] (Stream) No chunks available
2021-02-02 16-47-03.005 [Info] (Stream) No chunks available
2021-02-02 16-47-04.007 [Info] (Stream) No chunks available
2021-02-02 16-47-05.011 [Info] (Stream) No chunks available
2021-02-02 16-47-06.025 [Info] (Stream) No chunks available
2021-02-02 16-47-07.011 [Info] (Stream) No chunks available
2021-02-02 16-47-08.012 [Info] (Stream) No chunks available
2021-02-02 16-47-09.012 [Info] (Stream) No chunks available
2021-02-02 16-47-10.004 [Info] (Stream) No chunks available
2021-02-02 16-47-11.010 [Info] (Stream) No chunks available
2021-02-02 16-47-12.015 [Info] (Stream) No chunks available
2021-02-02 16-47-13.015 [Info] (Stream) No chunks available
2021-02-02 16-47-14.019 [Info] (Stream) No chunks available
2021-02-02 16-47-15.005 [Info] (Stream) No chunks available
2021-02-02 16-47-16.018 [Info] (Stream) No chunks available
2021-02-02 16-47-17.018 [Info] (Stream) No chunks available
2021-02-02 16-47-18.003 [Info] (Stream) No chunks available
2021-02-02 16-47-19.016 [Info] (Stream) No chunks available
```

**Comment by badaix** [comment]
Fixed c342b15, nightly build: https://github.com/badaix/snapcast/actions/runs/533416733

**Comment by tremby** [comment]
Seems fixed. Thank you.
